src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1076,7 +1076,7 @@ bool lv_vg_lite_path_check(const vg_lite_path_t * path)
         case VG_LITE_DRAW_FILL_PATH:
         case VG_LITE_DRAW_FILL_STROKE_PATH: {
                 /* Check end op code */
-                uint8_t end_op_code = VLC_GET_OP_CODE(end - fmt_len);
+                uint8_t end_op_code = LV_VG_LITE_PATH_GET_OP_CODE(end - fmt_len);
```

Comment:
I noticed there is a subtraction to an unsigned destination value, there is no issue if fmt_len is higher than end (which leads to wrapping values)?  Or this situation is guaranteed that will never happen?

Reply:
This won't be a problem. end is a pointer to the address after the last byte at the end of path. `fmt_len` indicates the unit length of path, which is within 4, so no errors can occur.
This code indicates that the last data of the path is fetched. The last data of the closed path should be `VLC_OP_END`.